### ISSUE TITLE
load pretrained model(trained in torch0.4) and got nan loss in torch1.2

### ISSUE THREAD

**Issue by ehion** [pytorch/pytorch#25243]
Is it possible to cause Nan loss in pytorch1.2 when i load pretrained model trained in pytorch0.4?
Because pytorch1.2 support sync-bn while pytorch0.4 does not.
I have tested just random init and did not load pretrained model and the nan loss did not appear
